cpp/include/Ice/Communicator.h

```diff
@@ -20,41 +20,37 @@ namespace Ice
     class LocatorPrx;
     class RouterPrx;
 
-    /// The central object in Ice. One or more communicators can be instantiated for an Ice application.
-    /// @see Logger
-    /// @see ObjectAdapter
-    /// @see Properties
-    /// @see ValueFactory
+    /// Represents the main entry point into the Ice runtime. You create a communicator with
+    /// {@link Ice::initialize(int&, const char*[], InitializationData)} or another `initialize` overload.
+    /// @see ::initialize(int&, const char*[], InitializationData)
+    /// @see ::initialize(InitializationData)
     /// @headerfile Ice/Ice.h
     class ICE_API Communicator final : public std::enable_shared_from_this<Communicator>
     {
     public:
         ~Communicator();
 
-        /// Destroy the communicator. This operation calls {@link #shutdown} implicitly. Calling {@link #destroy} cleans
-        /// up memory, and shuts down this communicator's client functionality and destroys all object adapters.
-        /// Subsequent calls to {@link #destroy} are ignored.
+        /// Destroys the communicator. This function calls #shutdown implicitly. Calling #destroy destroys all object
+        /// adapters, shuts down this communicator's client functionality, and cleans up memory. Subsequent calls to
+        /// #destroy are no-op.
+        /// @see CommunicatorHolder
         /// @see #shutdown
```

Comment:
There is already a link to shutdown in the description `#shutdown implicitly` we can remove this `@see`.